tools/metrics/grafana/dashboards/buildbuddy.json

```
@@ -8525,219 +8516,290 @@
       "type": "row"
     },
     {
-      "collapsed": true,
+      "collapsed": false,
```

nit: revert so that all panels stay collapsed on startup

```suggestion
      "collapsed": true,
```

(I'd like to have the script take care of this automatically at some point)